### ISSUE TITLE
【feature request】support resource pools across multiple cloud providers

### ISSUE THREAD

**Issue by LangDaoAI** [determined-ai/determined#5910]
Hi team,

Is there any plan to support resource pools across multiple cloud providers?

From https://docs.determined.ai/latest/introduction.html:

![image](https://user-images.githubusercontent.com/83859417/216305593-2f0fec3d-a653-4352-928d-a925e1a791ac.png)

Thanks!

**Comment by rb-determined-ai** [comment]
Is the feature request for one resource pool across multiple clouds, or multiple resource pools, one per cloud?

**Comment by LangDaoAI** [comment]
hi @rb-determined-ai I think that current system architecture can not support one resource pool across multiple clouds(the master in some cloud region has no way to communicate with dynamic agents from different cloud providers); On the other hand, about "multiple resource pools, one per cloud",    unless the master can communicate with each resource pool by some network.

I want to hear your some suggestions.

Thanks!

**Comment by rb-determined-ai** [comment]
You are right, we really can't offer one resource pool across multiple clouds.  Also, it would have severe impacts on training performance due to the network latency.

We are aware of the feature request for multiple resource pools, one per cloud.  At this time, it is not currently on our roadmap (planned 12 months out).

But I've made our product team aware of your request.

**Comment by LangDaoAI** [comment]
> You are right, we really can't offer one resource pool across multiple clouds. Also, it would have severe impacts on training performance due to the network latency.
> 
> We are aware of the feature request for multiple resource pools, one per cloud. At this time, it is not currently on our roadmap (planned 12 months out).
> 
> But I've made our product team aware of your request.

@rb-determined-ai about "multiple resource pools, one per cloud", I have the following consideration for the system architecture:

![multi masters](https://user-images.githubusercontent.com/83859417/216877570-c4575d73-f82f-4d85-bf84-51948f5c8658.png)

on the architecture, I have a private datacenter hosting master_proxy  which can route into master_aws and master_gcp, forming multiple resource pools, one per cloud, and webui is hosted the same as the master_proxy which can access the two postgreses from different cloud providers for the metadata.

As far as I know, current system architecture can not support  "multiple resource pools, one per cloud".

please correct my any problem, thanks!
